test/Microsoft.SqlTools.ManagedBatchParser.IntegrationTests/BatchParser/BatchParserTests.cs

```diff
@@ -232,15 +230,52 @@ public void VerifyLexerSetState()
                 }
                 executionResult = ScriptExecutionResult.Success;
             }
-            catch (Exception ex)
+            catch (Exception)
             {
-
                 executionResult = ScriptExecutionResult.Failure;
-
             }
             //  we doesn't expect any exception or testCase failures
             Assert.Equal<ScriptExecutionResult>(ScriptExecutionResult.Success, executionResult);
-             
+        }
+
+        // This test case is to verify that, Powershell's Invoke-SqlCmd handles ":!!if" in an inconsistent way
```

Comment:
Seems like the comment should say `in a consistent way` instead of `inconsistent way`.  But maybe I'm reading the text wrong.